README.md

```diff
@@ -32,7 +32,7 @@ Current implementation fully supports 37 HomeKit accessory/services.
 |  Air Quality Sensor               |     :white_check_mark:    |
 |  Audio Stream Management          |     :x:                   |
 |  Battery Service                  |     :white_check_mark:    |
-|  Camera RTP Stream Management     |     :x:                   |
+|  Camera RTP Stream Management     |      :white_check_mark:   |
```

Comment:
needs update from hap-java/hap-java 

And this text needs update:
Current implementation fully supports 37 HomeKit accessory/services.